### PR TITLE
Created examples folder

### PR DESCRIPTION
**Pull request by ryannguyen16**
# Description

created examples folder and started added examples

# Testing

# Pull Request Type

Please select the option(s) that are relevant to this PR.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Improvement (fixing a typo, updating readme, renaming a variable name, etc)

### CODE REVIEW

**Comment by nwithan8** [comment]
Looks good so far, will thoroughly review when it's ready.
Make sure your IDE is configured to add an empty line at the end of each file. There's some in here that don't have it (notice the red mark on GitHub diffs)

File: examples/insurances/CreateInsurance.cs [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+            }
+        }
+    },
+    { "reference", "InsuranceRef1" },
+    { "carrier", "USPS" },
+    { "tracking_code", "9400110898825022579493" },
+    { "amount", "100.00" }
+});
+
+
```
Comment (nwithan8) [inline examples/insurances/CreateInsurance.cs:27]: Delete the extra lines at the bottom of this file.

File: examples/insurances/RetrieveInsuranceList.cs [hunk 2]
```diff
@@ -0,0 +1,10 @@
+using EasyPost;
+
+EasyPost.ClientManager.SetCurrent("EASYPOST_API_KEY");
+
+listParams = new Dictionary<string, object>() {
```
Comment (nwithan8) [inline examples/insurances/RetrieveInsuranceList.cs:5]:
```suggestion
Dictionary<string, object> listParams = new Dictionary<string, object>() {
```
```suggestion
listParams = new Dictionary<string, object>() {
```

File: examples/pickups/CreatePickup.cs [hunk 3]
```diff
@@ -0,0 +1,14 @@
+using EasyPost;
+
+EasyPost.ClientManager.SetCurrent("EASYPOST_API_KEY");
+
+parameters = new Dictionary<string, object>() {
```
Comment (nwithan8) [inline examples/pickups/CreatePickup.cs:5]:
```suggestion
Dictionary<string, object> parameters = new Dictionary<string, object>() {
```

File: examples/trackers/RetrieveTrackerList.cs [hunk 4]
```diff
@@ -0,0 +1,10 @@
+using EasyPost;
+
+EasyPost.ClientManager.SetCurrent("EASYPOST_API_KEY");
+
+var listParams = new Dictionary<string, object>() {
```
Comment (nwithan8) [inline examples/trackers/RetrieveTrackerList.cs:5]:
```suggestion
Dictionary<string, object> listParams = new Dictionary<string, object>() {
```

**Review by nwithan8 — CHANGES_REQUESTED** [review]

**Comment by Justintime50** [comment]
No longer valid since these are moving to the `examples` repo.

**Pull request closed without merging**